lib/adobe/campaign/language.rb

```diff
@@ -0,0 +1,192 @@
+# frozen_string_literal: true
+module Adobe
+  module Campaign
+    class Language
+
+      def self.map_code(language_code)
+        return language_code if language_code.empty?
+
+        code = normalize language_code
+
+        return code if @adobe_language_codes.include?(code)
+        return code.gsub("-", "_") if @adobe_language_codes.include?(code.gsub("-", "_"))
+
+        if code.include?("-")
+          map_code code.rpartition("-").first
+        else
+          @adobe_language_codes.find{ |s| s.start_with? code}
+        end
+
+      end
+
+      def self.normalize(language_code)
+        code = language_code
+        code = code.downcase
+        code.gsub("_", "-")
+      end
+
+      private
+
+      @adobe_language_codes = [
```

Comment:
Should this be a constant instead? 
Where was this pulled from? 
Will this list change?

Reply:
This could be a constant. I started with a class variable (static), but was informed it was bad practice. I'll convert to constant.

Reply:
Could you add a comment as to where this list was found so that future maintainers can update this easier please?